src/main/resources/templates/laDigitalAssister/jobHourlyWage.html

```diff
@@ -9,7 +9,7 @@
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <p class="employer-name" th:text="${fieldData.get('employerName')}"></p>
+        <p class="grey-text" th:text="${fieldData.get('employerName')}"></p>
```

Comment:
fyi @lkemperman-cfa - changed the class to be more universal than employer name.